domain/src/mongo.js

```diff
@@ -19,6 +20,7 @@ function disconnect(db) {
 async function addItem(db, coll, item) {
     if (db === undefined) throw new Error('нет базы данных');
     let collection = db.collection(coll);
+    item = utility.clone(item);
```

Comment:
without 23 line, after `let res = await collection.insertOne(item)` our variable `item` has the property `_id` O_O